lib/hooks/useContributionsList.ts

```diff
@@ -1,16 +1,21 @@
 import useSWR from "swr";
+import useStore from "lib/store";
 
 interface PaginatedContributorsResponse {
   readonly data: DbContribution[];
   readonly meta: Meta;
 }
 
 const useContributionsList = (repoId: string, limit = "", orderBy = "") => {
+  const range = useStore(state => state.contributorRange);
```

Comment:
We want to keep the same range across the different tabs of the dashboard, so we can reuse the `range` property and have the other hooks revalidate when it has changed.